test/support/integration/teams_server.ex

```diff
@@ -105,7 +105,7 @@ defmodule Livebook.TeamsServer do
   # Private
 
   defp call_erpc_function(node, function, args \\ []) do
-    :erpc.call(node, Hub.Integration, function, args)
+    :erpc.call(node, Teams.Integration, function, args)
```

Comment:
Gah, now calling this `Teams` will be confusing.

Maybe we should alias `Teams.Integration` to `TeamsServer` and use `TeamsServer` throughout? We can also rename `Teams.Integration` to `TeamsServer` in the "teams" app itself. Thoughts?

Reply:
I like the idea of renaming it to TeamsServer

Reply:
So let's do that :)